src/sentry_plugins/github/endpoints/webhook.py

```diff
@@ -36,7 +36,7 @@ def __call__(self, organization, event):
         repo = Repository.objects.get_or_create(
```

Comment:
As a part of this, let's change this to just a `get()` and return a 404 when the repository doesn't exist. The lazy creation here is buggy at best anwyays since we can't populate a name. This will allow returnign back something useful to GitHub and a manual re-deliver of failed webhooks can be done.